### DISCUSSION TITLE
uBlock Origin blocks websockets on Firefox

### DISCUSSION THREAD

**Discussion by trmckay** [advplyr/audiobookshelf#500] [General]
I don't think this is a bug. But, it was a major headache for me to find. Should we add a warning somewhere in the docs?

**Comment by advplyr** [comment]
I'm not sure what this is, can you elaborate?

**Reply by trmckay** [reply]
Of course. It's a popular ad/tracker/miner blocker browser extension. I believe it mistakenly flags the websocket as mistaken/unwanted. It's tough to say whether this is due to the content of the page, my domain, or something else. But regardless I figured it was worth mentioning somewhere.

**Reply by advplyr** [reply]
Do you have steps to resolve the issue? I'll add it to the docs but I'm not sure what to put.
